Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            static void bmiCalculator()
+            {                
+                string firstName = Checks.PromptString("What is your first name?");
```

Comment:
Call PromptString function in checks. PromptString function should be in YOUR OWN static class. Checks class is there to simply allow you to name your function and place them in whatever classes you want.